front_end/.zshrc

```diff
@@ -0,0 +1 @@
+export PATH="$PATH:$HOME/.yarn/bin"
```

Comment:
Our project should not include code that forces a change in the user's environment.  We *can* add this to the README if this is necessary, but since yarn is a replacement for npm, I suspect that we should ask developers to install it globally, in the README (which is what [facebook recommends](https://code.facebook.com/posts/1840075619545360)):
```txt
npm install -g yarn
```